### PR TITLE
add deploy octobot button and update star history

### PR DESCRIPTION
**Pull request by GuillaumeDSM**
image

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -78,8 +79,10 @@ GEM
       tzinfo (>= 1.0.0)
     unicode-display_width (1.8.0)
     wdm (0.1.1)
+    webrick (1.7.0)
 
 PLATFORMS
+  x64-mingw-ucrt
```
Comment (Herklos) [inline Gemfile.lock:85]: 😲

File: index.html [hunk 2]
```diff
@@ -17,12 +17,21 @@
...
-                       href="{% tl demo_try %}"
-                       class="btn btn-lg call-to-action-button">
-                        {% t landing_page.try_octobot %}
-                    </a>
+                <div class="row w-100 offset-2">
+                    <div class="col-md-4">
+                        <a id="home-first-try-now"
+                            href="{% tl demo_try %}"
+                            class="btn btn-lg call-to-action-button">
+                            {% t landing_page.try_octobot %}
```
Comment (Herklos) [inline index.html:25]: Could we make this button bigger?

File: Gemfile.lock (hunk 1 above)
Comment (GuillaumeDSM) [reply Gemfile.lock:85]: windows stuff x)

Comment (Herklos) [reply Gemfile.lock:85]: 😂

**Review by Herklos — APPROVED** [review]
👍

**Pull request merged**